### ISSUE TITLE
Musical examples?

### ISSUE THREAD

**Issue by bcmoseley** [openmusictheory/openmusictheory.github.io#29]
Do we have a general consensus as to whether pages should have musical examples? A lot of the sonata form pages don't. Should we keep those pages text only or should we be looking to add graphics?

**Comment by bcmoseley** [comment]
If not graphics, maybe just a link to a recording?

**Comment by kshaffer** [comment]
I like the idea of having at least one audio example for each concept/chapter throughout. I'd prefer not to get the text bogged down in explaining examples, since that will detract from its utility as a quick reference during student work. I think the pop/rock harmony model is good: define/explain concept, give one or two examples of that concept, but don't spend a lot of time unpacking it; it's just a model. What do you think?

**Comment by bcmoseley** [comment]
I agree. For those pages that do have a quick example, have we decided if we'd like them to be typeset?
